tests/flow/test_pending_queries_limit.py

```diff
@@ -8,19 +9,43 @@
 # 3. test overflowing the server when there's no limit
 #    expect not to get any exceptions
 
+con = None
```

Comment:
Why do we need a global `con` ?

Reply:
@swilly22 to hold the connection from the init function till the working functions being called